packages/docs-site/src/content/docs/docs/guides/schema.mdx

```diff
@@ -402,18 +398,20 @@ export default defineDmnoService({
 });
 ```
 
-Internally, we even wrap the static values and inline functions into resolvers, so that we can always display some additional metadata about _how_ the value is being set - even without actually resolving it. These resolvers can also return more resolvers, and have a notion of multiple "branches" built in, leading to some very powerful composition capabilities.
+Internally, we even wrap the static values and inline functions into resolvers, so that we can always display some additional metadata about _how_ the value is being set - even without actually resolving it. These resolvers can also return more resolvers, and have a notion of multiple branches built in, leading to some very powerful composition capabilities.
 
-An quick example to illustrate using our built-in `switchByNodeEnv` resolver which selects a value based on the current value of `NODE_ENV` within your config.
+An quick example to illustrate using our built-in [`switchByNodeEnv`](/docs/reference/helper-methods/#switchbynodeenv) resolver which selects a value based on the current value of `NODE_ENV` within your config.
 
 ```ts
 export default defineDmnoService({
   pick: ['NODE_ENV'],
   schema: {
     SOME_API_KEY: {
       value: switchByNodeEnv({
-        // some dev env key, so not a problem to expose
+        // static values for pre-prod, so not a problem to include here
         _default: 'abc123',
+        development: 'dev123',
```

Comment:
I think I'd either do `_default` or `development` but not both, since development is usually the default NODE_ENV value. You could do _default, test, staging, production?

Reply:
maybe `development` since most people would be used to that value in `NODE_ENV`? I can add a comment about `_default` too